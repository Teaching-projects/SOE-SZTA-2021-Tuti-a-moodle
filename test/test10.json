{
    "name": "Max",
    "lore": "Description for Max",
    "health_points": 85,
    "attack_cooldown": 1,
    "damage": 40
}
