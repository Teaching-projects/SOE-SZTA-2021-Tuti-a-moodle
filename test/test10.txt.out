====================
Bob - HP: 90.0, DMG: 60.0, DEF: 0.0
Bob - HP: 90.0, DMG: 60.0, DEF: 0.0
====================
The Battle begins.
Bob and Bob clash!
====================
The Battle is over.
Both heroes are dead!